ArduCopter/mode.cpp

```diff
@@ -688,9 +688,9 @@ Copter::Mode::AltHoldModeState Copter::Mode::get_alt_hold_state(float target_cli
 // these are candidates for moving into the Mode base
 // class.
 
-float Copter::Mode::get_surface_tracking_climb_rate(int16_t target_rate, float current_alt_target, float dt)
```

Comment:
Consider taking this opportunity to change the namespace of the implementation to Copter::Mode; I think `copter.rangefinder_state` may be the big change there as we don't take a reference to that one when constructing the mode object.